velox/common/file/FileSystems.h

```diff
@@ -49,7 +49,9 @@ struct FileOptions {
 class FileSystem {
  public:
   FileSystem(std::shared_ptr<const Config> config)
-      : config_(std::move(config)) {}
+      : config_(std::move(config)) {
+    VELOX_CHECK_NOT_NULL(config_)
```

Comment:
Linux filesystem does not need a config. We should remove this check.

Reply:
thank you, already removed.